src/channel.ts

```diff
@@ -1263,6 +1263,9 @@ export class Channel<
    */
   async banUser(targetUserID: string, options: BanUserOptions<UserType>) {
     this._checkInitialized();
+    if (options?.user_id !== undefined) {
```

Comment:
I don't think you need it here, since it gets dispatched to the client banUser()